ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -88,6 +88,27 @@
       data:
         github_bot_token.txt: "{{ lookup('file', operator_pipeline_github_bot_token, rstrip=False) | b64encode }}"
 
+- name: Create Operator pipeline github webhook secret
+  no_log: yes
+  k8s:
+    state: present
+    force: yes
+    namespace: "{{ oc_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: Secret
+      type: opaque
+      metadata:
+        name: github-webhook-secret
+        labels:
+          app: operator-pipeline
+          suffix: "{{ suffix }}"
+          env: "{{ env }}"
+      data:
+        webhook-secret-preprod: "{{ lookup('file', operator_pipeline_webhook_secret_preprod, rstrip=False) | b64encode }}"
```

Comment:
There should be only one secret based on the env where the webhook + secret is deployed.